packages/modular-scripts/src/build.ts

```diff
@@ -465,28 +472,35 @@ async function makeBundle(
     // TODO: what of 'bin' fields?
     main: preserveModules
```

Comment:
What’s the flow for using preserve modules? What’s the right way for consumers to use this flag? I would imagine that it would actually make more sense as a flag in the modular metadata in package.json for the package so that this flag is abstracted from the commands, otherwise I foresee people using it differently within a team when working.

Reply:
Yeah if you see the list of TODOs at the bottom of this file, having it as metadata in package.json is a line item. Considering this an unreleased feature for now so I haven't documented a flow yet. Want to use it at JPM and our own repo and then make a call whether it makes sense.